Neos.Neos/Documentation/References/NeosFusionReference.rst

```diff
@@ -484,6 +484,27 @@ Example::
 
 .. note:: Most of the time this can be simplified by directly assigning the value instead of using the ``Value`` object.
 
+.. _Neos_Fusion__Match:
+
+Neos.Fusion:Match
+-----------------
+
+Matches the given subject to a value
+
+:@subject: (string, **required**) The subject to match
+:@default: (mixed) The default to return when no match was found
+:[key]: (mixed) Definition list, the keys will be matched to the subject and their value returned.
```

Comment:
if the key is of type "mixed" (which I would like very much) we should add some test cases for this.
Can it be numbers and even arrays?

Reply:
ah, hang on. the key itself has to be a `string` I assume. Nevermind